src/actions/index.js

```diff
@@ -0,0 +1,24 @@
+let newId = 2; //only for add id to card
+
+const addCard = (name, email) => ({
+  type: "ADD_CARD",
+  name: name,
+  email: email,
+  id: ++newId
+});
+
+const deleteCard = id => ({
+  type: "DELETE_CARD",
+  id
+});
+
+const toggleAddCard = {
+  type: "TOGGLE_ADD_CARD",
+};
+
+const filter = filterText => ({
+  type: "FILTER",
+  filterText
+});
+
+export {addCard ,deleteCard, toggleAddCard, filter}
```

Comment:
What about the situation when there will be much more actions?
You can export them separately and have the same effect as you have now.
```
export const deleteCard = id ={
...
export const filter = filterText => ({
```

Reply:
ok its much simpler